src/ContaoFramework.php

```diff
@@ -167,6 +176,56 @@ public function initialize()
     }
 
     /**
+     * Creates a new instance of a given class.
+     *
+     * @param string $class Fully qualified class name.
+     * @param array $args Constructor arguments.
+     *
+     * @return mixed
+     */
+    public function createInstance($class, $args = [])
```

Comment:
The new methods should only be added to the new `ContaoFramework` class, shouldn't they?